docs/cub/tuning.rst

```diff
@@ -382,11 +386,129 @@ If all those three values are larger than 1.0, the variant is strictly better th
 If only the mean or max are larger than 1.0, the variant may perform better in most runtime workloads, but regress in others.
 This information can be used to change the existing tuning policies in CUB.
 
+
+Variant plots
+--------------------------------------------------------------------------------
+
+The reported score for a tuning aggregates the performance across all runtime workloads.
+Even though the min, mean and max score are given as well, it may be necessary to view the distribution of scores across variants.
+
 ..
-    TODO(bgruber): the following is outdated:
+    TODO(bgruber): the following is outdated and should be rewroted
```

Comment:
This should have been `reworded`, but it could also have been `rewritten`. It's a mix of both words and I like it :D